spec/features/recipe_ingredients/create_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe "the recipe ingredient new page" do
+  before(:each) do
+      @beef_pasta = Recipe.create(name: "Beef Pasta", complexity: 2, genre: "Italian")
+      @ground_beef = @beef_pasta.ingredients.create(name: "Ground Beef", cost: 4)
+      @garlic = @beef_pasta.ingredients.create(name: "@garlic", cost: 2)
+      @tomato_paste = @beef_pasta.ingredients.create(name: "Tomato Paste", cost: 1)
+  end
+
+  it "renders the new form" do
+    visit "/recipes/#{@beef_pasta.id}/new"
+
+    expect(find("form")).to have_content("Ingredient Name")
+    expect(find("form")).to have_content("Ingredient Cost")
+  end
```

Comment:
I think it might be easier to find the form once and store that in a variable so that you don't have to keep finding it each time.

```ruby
form = find("form")

expect(form).to have_content("Ingredient Name")
expect(form).to have_content("Ingredient Cost")
```

Beyond that, I think I would opt for some more specific capybara matchers like:

```ruby
expect(form).to have_field(:name)
expect(form).to have_field(:cost)
```